packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -51,6 +54,33 @@ export function SidebarNavigationItemGlobalStyles( props ) {
 	);
 }
 
+function SidebarNavigationScreenGlobalStylesContent() {
+	const { storedSettings } = useSelect( ( select ) => {
+		const { getSettings } = unlock( select( editSiteStore ) );
+
+		return {
+			storedSettings: getSettings( false ),
+		};
+	}, [] );
+
+	// Wrap in a BlockEditorProvider to ensure that the Iframe's dependencies are
+	// loaded. This is necessary because the Iframe component waits until
+	// the block editor store's `__internalIsInitialized` is true before
+	// rendering the iframe. Without this, the iframe previews will not render
+	// in mobile viewport sizes, where the editor canvas is hidden.
+	return (
+		<BlockEditorProvider
+			settings={ storedSettings }
+			onChange={ noop }
+			onInput={ noop }
+		>
+			<div className="edit-site-sidebar-navigation-screen-global-styles__content">
```

Comment:
Just checking, is the `<div />` necessary here too?

Reply:
It's not _necessary_, but I copied the same approach as over in the Navigation screen, because if we're wrapping components within `content` in this screen, it felt like it might be good to have a container specific to this screen, particularly because we'll likely be adding other components to this screen in follow-ups (e.g. revisions potentially).

Happy to remove it for the moment if you don't think it's warranted for this PR! 🙂